Create 2020-11-16-mounts.md (#120)
tomeshnet/toronto-community-network: operations/hardware-budgets/2020-11-16-mounts.md

# 2020-11-16 Hardware Budget for Toronto Community Network

## Purpose

_What is the hardware for? If this is related to a particular deployment and require approval by a specific date, please mention it here._

> Purhcase 5 mounts to have in stock and deply as needed.

Reviewer: ```suggestion
Purchase 5 mounts to have in stock and deploy as needed.
```

## Hardware List

_A list of all hardware items and the prices from specific vendors we plan to order from._

| Item        | Vendor     | Price  | Currency | Quantity | 
|:------------|:-----------|-------:|---------:|---------:|
| CS-18274(1)    | Infinity Cables|  180   |   CAD    |        5 |
| Cinder Blcks| Home Depot |    5   |   CAD    |        20 |

> (1) CS-18274: Custom Southwinds Engineering Roof Mount as per Cisco specs FOB Flesherton, Ontario facility.

Reviewer: ```suggestion
(1) CS-18274: Custom Southwinds Engineering Roof Mount as per Cisco specs for the Digital Canopy project. FOB Flesherton, Ontario facility.
```

## Budget Requested

_The Administrators acknowledge that final reimbursement amounts may differ due to unforeseen costs like duties and exchange rates. This should be a best-effort estimate in Canadian Dollars._

| Budget    | Estimate (CAD) |
|:----------|---------------:|
| Hardware  |        1000.00 |
| Shipping  |          50.00 |
| Duties    |           0.00 |
| Taxes     |         136.50 |
| **Total** |        1186.50 |

## Sign-off

_The sign-off should include at least one experienced member in the Network working group._

Members of Network Planning, Design and Operations:
- @yurkowashere
- @name

Grant Administrators:
- @YurkoWasHere 

> - @name

Reviewer: ```suggestion
- @benhylau
```
